### PR TITLE
Adding a script to classify data and add dependencies 

### PR DESCRIPTION
**Pull request by Melviyn**
Adding a script to classify data and add dependencies  to respect OSGI structure

### CODE REVIEW

File: src/main/java/org/orbisgis/mapuce/randomForest/BuildModel.java [hunk 1]
```diff
@@ -79,12 +79,11 @@ public BuildModel(String path,String pathModelFile, int classIndex) throws Excep
...
         header.delete();
         write[1] = header;
-        
+
         weka.core.SerializationHelper.writeAll(new FileOutputStream(pathModelFile), write);
 
-        
-        
         pathModel = pathModelFile;
+        System.out.println(pathModelFile);
```
Comment (ebocher) [inline src/main/java/org/orbisgis/mapuce/randomForest/BuildModel.java:86]: Please remove this

File: pom.xml [hunk 2]
```diff
@@ -40,6 +40,35 @@
...
+                            netlib-native_system-win-i686;scope=compile;inline=true,
+                            netlib-native_system-win-x86_64;scope=compile;inline=true,
+                            netlib-native_ref;scope=compile;inline=true,
+                            netlib-native_system;scope=compile;inline=true,
+                            core;scope=compile;inline=true,
+                            jniloader;scope=compile;inline=true,
+                            ant-launcher;scope=compile;inline=true,
+                            ant;scope=compile;inline=true,
+                            xz;scope=compile;inline=true
+                        </Embed-Dependency>
```
Comment (ebocher) [inline pom.xml:71]: Required ? What about the OrbisGIS dependencies repo ?

Comment (SPalominos) [inline pom.xml:71]: It seems that the change has been done in is github repo, in the branch dev : [dev/pom.xml](https://github.com/Melviyn/MApUCE_tools/commit/dc4b3455d327405b98bcc9111c17e56b54ae8469)

**Comment by SPalominos** [comment]
This seems to be an outdated PR. Maybe it should be closed, and a new one should be open with the branch dev of @Melviyn's github repo : Melviyn/MApUCE_tools@dc4b3455d327405b98bcc9111c17e56b54ae8469

**Comment by Melviyn** [comment]
I close this PR and I opened a new one with the right branch  ?

**Comment by SPalominos** [comment]
It will be easier that way.
Close this PR, and open a new one from Melviyn:dev to orbisgis:master. So we'll be able to comment your last code version.

**Pull request closed without merging**